File: python/UberVan.py
from Car import Car
from Account import Account

class UberVan(Car):
    typeCarAccepted: list
    seatsMaterial: list
    
    def __init__(self, license: str, driver: Account, typeCarAccepted: list, seatsMaterial: list) -> None:
        super().__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial

File: js/UberVan.js
import { Car } from "./Car";

class UberVan extends Car{
    constructor(license, driver, typeCarAccepted, seatsMaterial){
        super(license, driver)
        this.typeCarAccepted = typeCarAccepted
        this.seatsMaterial = seatsMaterial
    }
}

export {UberVan}